packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -76,18 +81,21 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
   useLocalVideoStartTrigger(!!props.isVideoStreamOn);
   const VideoGalleryMemoized = useMemo(() => {
     return (
-      <VideoGallery
-        {...videoGalleryProps}
-        localVideoViewOptions={localVideoViewOptions}
-        remoteVideoViewOptions={remoteVideoViewOptions}
-        styles={VideoGalleryStyles}
-        layout="floatingLocalVideo"
-        showCameraSwitcherInLocalPreview={props.isMobile}
-        localVideoCameraCycleButtonProps={cameraSwitcherProps}
-        onRenderAvatar={onRenderAvatar}
-      />
+      <>
+        <Announcer announcementString={announcerString} ariaLive={'assertive'} />
```

Comment:
```suggestion
        <Announcer announcementString={announcerString} ariaLive={'polite'} />
```
?

Reply:
leftovers from some testing oops. changed back to polite